lib/src/main/java/net/ypresto/androidtranscoder/utils/AudioConversionUtils.java

```diff
@@ -0,0 +1,69 @@
+package net.ypresto.androidtranscoder.utils;
+
+import java.nio.ShortBuffer;
+
+public class AudioConversionUtils {
+
+    public interface Remixer {
```

Comment:
Please place this to `net.ypresto.androidtranscoder.engine`, as it is not util class but core logic.